src/pages/case-study/doctorlink-setting-up-a-design-system.js

```diff
@@ -0,0 +1,689 @@
+    tablet && tablet.top ? theme.space[tablet.top] : null};
+  padding-bottom: ${({ theme, tablet }) =>
+    tablet && tablet.bottom ? theme.space[tablet.bottom] : null};
+`}
+
+${breakpoint('desktop')`
+  padding-top: ${({ theme, desktop }) =>
+    desktop && desktop.top ? theme.space[desktop.top] : null};
+  padding-bottom: ${({ theme, desktop }) =>
+    desktop && desktop.bottom ? theme.space[desktop.bottom] : null};
+`}
+`
+
+const ReversableDiv = styled.div`
+  display: flex;
+  flex-direction: column-reverse;
+
+  ${breakpoint('smallTablet')`
+    flex-direction: column;
+  `}
+`
+
+const addPaddings = (top, bottom) => ({
+  top,
+  bottom
+})
+
+const getImage = (blockImages, index) => blockImages && blockImages[index]
+
+const normalise = (arr = []) => {
+  const normalised = arr.map(
+    ({ genericBlockText, genericBlockImages, ...props }) => ({
+      image: getImage(genericBlockImages, 0),
+      text: genericBlockText && genericBlockText.genericBlockText,
+      ...props
+    })
+  )
+  return normalised[0]
+}
+
+const normaliseImages = ({ genericBlockImages = [] }) => genericBlockImages
+
+const shouldRender = data => data && data.length
+
+const IndexPage = props => {
+  const {
+    data: { contentfulNonTemplatedCaseStudyV2: caseStudy },
+    location
+  } = props
+
+  const {
+    genericBlock1: data1,
+    genericBlock2: data2,
+    genericBlock3: data3,
+    genericBlock4: data4,
+    genericBlock5: data5,
+    genericBlock6: data6,
+    genericBlock7: data7,
+    genericBlock8: data8,
+    genericBlock9: data9,
+    genericBlock10: data10,
+    genericBlock11: data11,
+    genericBlock12: data12,
+    genericBlock13: data13,
+    genericBlock14: data14,
+    genericBlock15: data15,
+    genericBlock16: data16,
+    genericBlock17: data17,
+    genericBlock18: data18,
+    genericBlock19: data19,
+    genericBlock20: data20,
+    genericBlock21: data21,
+    genericBlock22: data22,
+    genericBlock23: data23,
+    genericBlock24: data24,
+    genericBlock25: data25,
+    genericBlock26: data26,
+    genericBlock27: data27,
+    genericBlock28: data28,
+    genericBlock29: data29,
+    genericBlock30: data30,
+    genericBlock31: data31
+  } = caseStudy
+
+  return (
+    <Layout location={location}>
+      <Head
+        page={{
+          ...caseStudy,
+          socialLogo:
+            'https://www.yld.io/static/logo_animated-832020608244057f6a9d73e80994ac4a.gif'
+        }}
+      />
+      <CaseStudyHero caseStudy={caseStudy} />
+
+      <GreyBackground>
+        <Grid>
+          {/* DoctorLink is a healthcare technology company with a mission to simplify the route to health and wellbeing for patients globally... */}
+          {shouldRender(data1) && (
+            <BlockRow
+              mobile={addPaddings('4', '4')}
+              smallTablet={addPaddings('4', '4')}
+              tablet={addPaddings('4', '4')}
+              desktop={addPaddings('6', '6')}
+              widthPerc="75%"
```

Comment:
please remove all widths from Row

Reply:
Using the `<Statement>` component solved that issue for us. Resolved.